consensus/protocol.go

```diff
@@ -51,26 +48,32 @@ type Protocol struct {
 	Primary bool
 }
 
-// Broadcaster defines the interface to enqueue blocks to fetcher and find peer
+// Protocol Manager defines the interface to enqueue blocks to fetcher, find peer
```

Comment:
Protocol manager?

Reply:
This interface is for this object:  https://github.com/celo-org/celo-blockchain/blob/sentry_v1/eth/handler.go#L70

I was intending to change the interface name as well (and will do so).